github-link.php

```diff
@@ -105,5 +108,18 @@ function GHL_plugin_link( $actions, $plugin_file, $plugin_data, $context ) {
         }
     }
 
+
+    // if from wordpress.org, show that now.
+    if ( ! empty( $plugin_data['url']) && strstr($plugin_data['url'],'wordpress.org') !== FALSE) {
+        $icon = "icon/wordpress_org_32x32.jpg";
+        $new_action = array ('wordpress_org' => sprintf(
+            '<img src="%s" style="vertical-align:-3px" height="16" width="16" alt="%s" />',
+            plugins_url( $icon, __FILE__ ),
+            "A wordpress.org plugin."
```

Comment:
I've never done that - I wouldn't even know where to start, but I replaced the previous text with __(...)